miscellaneous/build_templates/analysis.yml

```diff
@@ -0,0 +1,190 @@
+parameters:
+  language: ''
+  lib: false
+  skipBinSkim: false # Tells the analysis pipeline not to run binskim as it doesn't work for some samples
```

Comment:
Minor, but I would consistently capitalize BinSkim, Polaris, BlackDuck, and Python